neutromeratio/parameter_gradients.py

```diff
@@ -3,57 +3,189 @@
 import numpy as np
 import torch
 from pymbar import MBAR
+import logging
 from pymbar.timeseries import detectEquilibration
 from simtk import unit
+from tqdm import tqdm
 
 from neutromeratio.ani import AlchemicalANI
-from neutromeratio.constants import temperature, kT
+from neutromeratio.constants import kT, hartree_to_kJ_mol
+
+logger = logging.getLogger(__name__)
 
 
 class FreeEnergyCalculator():
     def __init__(self,
                  ani_model: AlchemicalANI,
                  ani_trajs: list,
                  potential_energy_trajs: list,
-                 lambdas,
+                 lambdas:list,
+                 n_atoms:int,
+                 per_atom_stddev_treshold:float=0.5,
                  max_snapshots_per_window=50,
                  ):
+        
+        """
+        Uses mbar to calculate the free energy difference between trajectories.
+        Parameters
+        ----------
+        ani_model : AlchemicalANI
```

Comment:
Is there a way to assert that the `ani_model` argument uses a linear interpolation between lambda=0 and lambda=1? Implementation assumes this

Reply:
good point! will add a parameter that needs to be checked.